### PR TITLE
Add MGBA Standalone

### PR DESCRIPTION
**Pull request by WingofaGriffin**
Diff to dev got messed up somehow so I'm based to beta for now.

Standalone mgba support;

- [x] MGBA Controller Config
- [x] SRM Scraper
- [x] Shell Script
- [x] Saves, States, Storage
- [x] Hotkeys
- [x] Convert Flatpak to AppImage
- [ ] Make sure image shows up in the GUI

MGBA allows from controller mapped hotkeys, but only for single button AFAIK, not combos. Set some other settings like 5x upscaling.

Bios not created as most users likely won't need it as the reverse engineered one works fine.

### CODE REVIEW

File: configs/steam-rom-manager/userData/userConfigurations.json [hunk 1]
```diff
@@ -1546,6 +1546,68 @@
...
+      "replaceDiacritics": true,
+      "removeCharacters": true,
+      "removeBrackets": true
+    },
+    "executable": {
+      "path": "/run/media/mmcblk0p1/Emulation/tools/launchers/mgba.sh",
+      "shortcutPassthrough": false,
+      "appendArgsToExecutable": false
+    },
+    "parserId": "164785621855061650",
```
Comment (WingofaGriffin) [inline configs/steam-rom-manager/userData/userConfigurations.json:1597]: tbh no idea how to define this

**Comment by rawdatafeel** [comment]
Icons for the AppImage Desktop file: https://github.com/mgba-emu/mgba/tree/master/res

**Comment by Godsbane** [comment]
Tyki, can you rebase on the current beta please?

**Comment by WingofaGriffin** [comment]
I think I got it done. There were some weird rebase things that went on, but should be resolved now

File: tools/chdconv/chddeck.sh [hunk 2]
```diff
@@ -176,7 +176,7 @@ if [ $ans -eq 0 ]; then
 		if [[ " ${csofolderWhiteList[*]} " =~ " ${romfolder} " ]]; then
 			find "$romsPath/$romfolder" -type f -iname "*.iso" | while read -r f; do
 				echo "Converting: $f"
-				ciso 9 "$f" "${f%.*}.cso" && rm -rf "$f"
+				ciso 9 "$f" "${f%.*}.cso" >"$HOME/.config/EmuDeck/chdtool.log" && rm -rf "$f"
```
Comment (Godsbane) [inline tools/chdconv/chddeck.sh:179]: Don't redirect to the log file, since it's set up at the start w tee

**Comment by WingofaGriffin** [comment]
don't know how that made it in there. done.

**Comment by Godsbane** [comment]
> don't know how that made it in there. done.

Happens lol.

I'll review in detail tonight

**Pull request merged**